staging/src/k8s.io/pod-security-admission/test/fixtures.go

```diff
@@ -26,20 +26,46 @@ import (
 	"k8s.io/utils/pointer"
 )
 
-// minimalValidPods holds minimal valid pods per-level per-version.
+// minimalValidOSNeutralPods holds minimal valid OS neutral pods per-level per-version.
 // To get a valid pod for a particular level/version, use getMinimalValidPod().
-var minimalValidPods = map[api.Level]map[api.Version]*corev1.Pod{}
+var minimalValidOSNeutralPods = map[api.Level]map[api.Version]*corev1.Pod{}
+
+// minimalValidLinuxPods holds minimal valid linux pods per-level per-version.
+// To get a valid pod for a particular level/version, use getMinimalValidPod().
+var minimalValidLinuxPods = map[api.Level]map[api.Version]*corev1.Pod{}
+
+// minimalValidWindowsPods holds minimal valid Windows pods per-level per-version.
+// To get a valid pod for a particular level/version, use getMinimalValidPod().
+var minimalValidWindowsPods = map[api.Level]map[api.Version]*corev1.Pod{}
 
 func init() {
-	minimalValidPods[api.LevelBaseline] = map[api.Version]*corev1.Pod{}
-	minimalValidPods[api.LevelRestricted] = map[api.Version]*corev1.Pod{}
+	initMinimalValidPods()
+}
+
+func addLinux(pod *corev1.Pod) *corev1.Pod {
+	copyPod := pod.DeepCopy()
+	copyPod.Spec.OS = &corev1.PodOS{Name: corev1.Linux}
+	return copyPod
+}
+
+func addWindows(pod *corev1.Pod) *corev1.Pod {
+	copyPod := pod.DeepCopy()
+	copyPod.Spec.OS = &corev1.PodOS{Name: corev1.Windows}
+	return copyPod
+}
```

Comment:
nit: to minimize diff:
* move addLinux/addWindows above init()
* declare the minimal valid pods directly in init() (don't need a separate initMinimalValidPods function)
* consider keeping `minimalValidPods` named as-is instead of renaming to `minimalValidOSNeutralPods` (can document these are OS-neutral in the godoc if you want)